<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarCatalog</class>
 <widget class="QMainWindow" name="CarCatalog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CarCatalog</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tw_cars">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="t_car_one">
     <attribute name="title">
      <string>Сторінка</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Producer:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Model Range:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of seats:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Color:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_producer">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_price">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_type">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_model_range">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_number_seats">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_color">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="t_car_two">
     <attribute name="title">
      <string>Сторінка</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Model Range:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_type_two">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_color_two">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_model_range_two">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Color:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Producer:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_price_two">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_producer_two">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_number_seats_two">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of seats:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="t_car_three">
     <attribute name="title">
      <string>Сторінка</string>
     </attribute>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Model Range:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_type_three">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_color_three">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_model_range_three">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Color:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Producer:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_price_three">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_producer_three">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_number_seats_three">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of seats:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>12</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="cb_filteringOptions"/>
     </item>
     <item>
      <widget class="QTextEdit" name="e_filteringField"/>
     </item>
     <item>
      <widget class="QPushButton" name="b_filter">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>327</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
